slack-plugin/src/main/kotlin/slack/gradle/SlackGradleUtil.kt

```diff
@@ -173,11 +173,11 @@ internal fun Project.jvmTargetVersion(): Int {
   return SlackProperties(this).jvmTarget
 }
 
-internal fun Project.getVersionsCatalog(): VersionCatalog {
-  return getVersionsCatalogOrNull() ?: error("No versions catalog found!")
+internal fun Project.getVersionsCatalog(name: String = "libs"): VersionCatalog {
+  return getVersionsCatalogOrNull(name) ?: error("No versions catalog found!")
 }
 
-internal fun Project.getVersionsCatalogOrNull(): VersionCatalog? {
+internal fun Project.getVersionsCatalogOrNull(name: String = "libs"): VersionCatalog? {
   return try {
     project.extensions.getByType<VersionCatalogsExtension>().named("libs")
```

Comment:
Use "name" instead of hard coded "libs"?

Reply:
derp

Reply:
[fee2643](https://github.com/slackhq/slack-gradle-plugin/pull/435/commits/fee2643ba0d182865797a8c13a38f4b786d432f7)